Cargo.toml

```diff
@@ -12,7 +12,7 @@ keywords = ["mac", "address", "network", "interface"]
 [dependencies]
 
 [target.'cfg(any(target_os = "linux", target_os = "macos"))'.dependencies]
-nix = "0.10.0"
+nix = "0.11.0"
 
 [target.'cfg(windows)'.dependencies]
-winapi = { version = "0.3.4", features = ["winerror", "ws2def"] }
\ No newline at end of file
+winapi = { version = "0.3.5", features = ["winerror", "ws2def"] }
```

Comment:
So I think that the version bumping should be left to a commit of its own -- I'd want to ensure that nothing has changed out from under it.